doc/user/content/cloud/update-deployments.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Update Deployments"
+description: "Update Materialize Cloud."
+menu:
+  main:
+    parent: "cloud"
+    weight: 6
+aliases:
+  - /cloud/upgrade-deployments
+---
+
+{{< cloud-notice >}}
+
+When a new version of Materialize Cloud is available, an **Update available** notice appears on the [Deployments](https://cloud.materialize.com/deployments) page. Click the notice and enter the requested confirmation to update the deployment.  We recommend updating a single deployment first and testing to ensure that your materializations and computations continue to work as expected.
+
+If you haven't updated a deployment to the latest version in two weeks, we will apply the new version automatically.
+
+If the update is unsuccessful, you will see a **Downgrade** button. Click this to restore the previous Materialize version.
```

Comment:
@msudberg 

We are not supporting a downgrade function in the Open Beta release via the UX. Users can downgrade if needed via API but otherwise will have to contact us to do so.

cc: @benesch